daemon/containerio/mem_buffer.go

```diff
@@ -12,7 +11,7 @@ func init() {
 }
 
 type memBuffer struct {
```

Comment:
Since we have changed the backend from memory buffer to pipe, the name `memBuffer` is no longer properly.

@ZouRui89 Please change to a more appropriate name.